src/dev-journal.js

```diff
@@ -1,23 +1,83 @@
 const client = require('./client')
 
 
-const isMonday = new Date().getDay() === 1
-const isWeekend = new Date().getDay() === 6 || new Date().getDay() === 0
-const subtractMe = isMonday ? 3 : 1
-const prevWorkDay = new Date(new Date().setDate(new Date().getDate() - subtractMe))
-  .toISOString()
-  .split('T')[0]
+const _getDay = (date) => (day) => new Date(date).toUTCString().toLocaleLowerCase().startsWith(day)
```

Comment:
this _may_ lead to problems in that [`Date.prototype.toUTCString()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toUTCString) is dependent on the platform implementation:

> Prior to ECMAScript 2018, the format of the return value varied according to the platform. The most common return value was an RFC-1123 formatted date stamp, which is a slightly updated version of RFC-822 date stamps.

it's still advisable to use methods like `getDay` or `getUTCDay` to retrieve a less ambiguously defined value to represent week days.